test/MongoUtil_test.py

```diff
@@ -123,7 +123,7 @@ def test_insert_one_ok(self):
         counter = mongo_util.get_hid_counter()
         mongo_util.insert_one(doc)
         new_counter = mongo_util.get_hid_counter()
-        self.assertEqual(new_counter, counter + 1)
```

Comment:
Do you have tests that insert multiple handles and check the autoincrement works?

Reply:
👍 just added a couple of tests for this.